mne/io/proj.py

```diff
@@ -779,14 +779,18 @@ def deactivate_proj(projs, copy=True, verbose=None):
 
 
 @verbose
-def make_eeg_average_ref_proj(info, activate=True, verbose=None):
+def make_eeg_average_ref_proj(info, activate=True, ch_dict=None,
```

Comment:
I would have gone with `ch_type=('eeg',)` as API and
default parameter

would that do the job for you?

Reply:
Yes, thank you for the quick review @agramfort! I changed it in this [commit](https://github.com/mne-tools/mne-python/pull/10656/commits/7e369d8d5aeba4100354107583d92129867d2369) 😊